features/steps/steps.py

```diff
@@ -58,6 +58,15 @@ class edge_use_error:
     def __str__(self):
         return f"On instance {fmt(self.inst)} the edge {fmt(self.edge)} was referenced {fmt(self.count)} times"
 
+@dataclass
+class representation_value_error:
+    inst: ifcopenshell.entity_instance
+    duplicate_values: str
```

Comment:
I think it's clearer to make duplicate_value singular.

So that you get:
- Inst has multiple Representations for Identifier Body: #123=asdasd #1233123=asdasd
- Inst has multiple Representations for Identifier Axis: #456=asdasd #456456456=asdasd
- ....